README.md

```diff
@@ -177,12 +171,6 @@ console.log(profile.relays)
 // prints: [wss://relay.damus.io]
 ```
 
-To use this on Node.js you first must install `node-fetch@2` and call something like this:
-
-```js
-nip05.useFetchImplementation(require('node-fetch'))
```

Comment:
Why did you remove this?

Reply:
adding it back. will resurface later in a separate PR. feels out of scope here

Reply:
[done](https://github.com/nbd-wtf/nostr-tools/compare/4fd83691968a7dacbc179fd08a571b4865329493..829e24b5bc65725b76172d764316e146e4ebe572)

using `cross-fetch` in test instead of `node-fetch` to support running same test in browser and node. [relevant commit](https://github.com/nbd-wtf/nostr-tools/pull/111/commits/0d577c67fd414ef26ea546efe7984a2e9849b5cb)